### ISSUE TITLE
Unable to use latitude/longitude with Point creation

### ISSUE THREAD

**Issue by philgresh** [adam-cowley/neode#82]
I get a ValidationError when trying to create a Project node and converting geolocation data to a Point. It looks like the validator converts the lat/lon input into x/y/srid correctly, but then fails when it expects to see latitude/longitude keys.

Am I using the model language incorrectly? Any help would be appreciated.

**Project Properties: (projectData)**
```javascript
{
  "id": "571b0144-0c49-4541-afe3-130f9edb3681",
// ...
  "geolocation": {
    "latitude": 37.795972,
    "longitude": -122.407994
  },
  "createdAt": "2019-09-20T05:09:47.564Z"
}
````

**Code: (nodejs 10.x in AWS Lambda)**
```javascript
const instance = new Neode
    .fromEnv()
    .with({
      Project: require('./models/Project')
    })
instance.create('Project', projectData)
````

**Project model file:**
```javascript
 
module.exports = {
  id: {
    type: 'uuid',
    primary: true,
    optional: false
  },
 // ...
  geolocation: {
    type: 'point',
    required: true,
    properties: {
      longitude: {
        type: 'number',
        required: true,
      },
      latitude: {
        type: 'number',
        required: true,
      },
    }
  },
  createdAt: {
    type: 'string',
    default: () => new Date().toISOString(),
  }
  
};


````

**Error:** 
```javascript
isJoi: true,
name: 'ValidationError',
details:
[
  {
    "message": "\"latitude\" is required",
    "path": [
      "geolocation",
      "latitude"
    ],
    "type": "any.required",
    "context": {
      "key": "latitude",
      "label": "latitude"
    }
  },
  {
    "message": "\"longitude\" is required",
    "path": [
      "geolocation",
      "longitude"
    ],
    "type": "any.required",
    "context": {
      "key": "longitude",
      "label": "longitude"
    }
  }
],
_object :
{ 
  id: '571b0144-0c49-4541-afe3-130f9edb3670',
  Point { srid: 4326, x: -122.407994, y: 37.795972, z: undefined },
// ...
  createdAt: '2019-09-20T05:09:47.564Z' 
},
annotate: [Function] }
````

**Comment by ngquangan** [comment]
I am having a similar problem. Have you found any solution yet?

**Comment by adam-cowley** [comment]
There are some problems with the validator tests - strange because I am using points in production without any issues.

I'll take a look over the next couple of days

**Comment by adam-cowley** [comment]
I've just published 0.3.4 - let me know if this fixes the issue.

**Comment by philgresh** [comment]
Thank you Adam, it works perfectly now!
